Gemfile

```
@@ -27,6 +27,7 @@ gem "warning"
 gem "pry"
 gem "excon"
 gem "jwt"
+gem 'pagerduty', '~> 2.1'
```

```suggestion
gem 'pagerduty'
```

Unless I know there's a pressing need for pagerduty 2.x, I usually let the gem definitions float and check the Gemfile.lock diff for anything "interesting" going on when upgrading. Otherwise, I am liable to never learn about 3.x.

---

Sometimes I use ">=" though, on rare occasion a series of gem interactions can result in a massive downgrade of a gem without the constraint, and "cannot solve" is superior to getting pagerduty 0.1.